src/assets/scripts/client/AppController.js

```diff
@@ -305,7 +305,7 @@ export default class AppController {
     updateViewControls() {
         const { current: airport } = AirportController;
 
-        this._eventBus.trigger(EVENT.MARK_DIRTY_CANVAS);
+        this._eventBus.trigger(EVENT.MARK_SHALLOW_RENDER);
```

Comment:
this should probably be `MARK_DEEP_RENDER` since we are potentially updating items that require a deep render

Reply:
As I went to do this, I found there actually _is no_ MARK_DEEP_RENDER event. Each thing that necessitates a deep render is handled through its own event, which is registered in the CanvasController to call its handler, each of which calls the appropriate re-render method...

Reply:
For example,
![image](https://user-images.githubusercontent.com/5103735/58748638-f0dcec00-8449-11e9-9c4b-e833d0719a28.png)

Reply:
Not sure how to proceed; we can open up an issue for it to get it figured out separately perhaps.

Reply:
Also worth noting (though you probably picked up on it), that all that happened was a rename of the existing event, so we can at least count on the behavior being the same as it was before the change, so there should be no new bugs from this. Doesn't mean it isn't wrong, but it does mean we won't break something new. :relieved:

Reply:
Biggest issue I had here was the name, it through me off a little. As long as it works I’m ok with it, we can circle back later.